examples/postgres/docker-compose.yaml

```diff
@@ -0,0 +1,40 @@
+version: "3.7"
+services:
+
+  proxy:
+    image: envoyproxy/envoy-dev:latest
+    volumes:
+      - ./envoy.yaml:/etc/envoy.yaml
```

Comment:
you will need to mount this file a docker image with a Dockerfile - like in the mysql example, and make sure its readable (the envoy proc runs as non-root in the container, CI checks to ensure this works where `umask` = `027`)

this is why verify.sh is failing only on CI

Reply:
Oh now I understand the reason why the mysql example uses a separated Dockerfile. Fixed!